start_ocean.sh

```diff
@@ -12,6 +12,12 @@ export BRIZO_ENV_FILE="${DIR}/brizo.env"
 DIR="${DIR/ /\\ }"
 COMPOSE_DIR="${DIR}/compose-files"
 
+# Default versions of Aquarius, Brizo, Keeper Contracts and Pleuston
+export AQUARIUS_VERSION=${AQUARIUS_VERSION:-v0.1.10}
```

Comment:
These versions are not right, see: https://github.com/oceanprotocol/barge/issues/127#issuecomment-479445626

Reply:
Yes I was waiting for there to be a stable set of versions working together. Now that one exists, I will update those version numbers.